src/views/GovernanceTaskList/index.scss

```diff
@@ -86,3 +86,9 @@
     color: #005ea2;
   }
 }
+
+.grid-row {
+  display: flex;
```

Comment:
I'm surprised you had to add this to the USWDS class. Can we take a look at the grid and see if we can solve this out of the box? Or, we might want to add our own css class name to tackle this. Given how we're doing imports, it's possible for the style of this component to impact other rows on the page.